### ISSUE TITLE
ARP Accessibility Testing: Submissions page + 526EZ

### ISSUE THREAD

**Issue by oddball-lindsay** [department-of-veterans-affairs/va.gov-team#126268]
This ticket is to perform accessibility testing (the following checklist) for the following pages:
1. Submissions page
2. 526EZ submissions pathway 

With the following OS/screen reader/browser combinations:

1. MacOS, Safari, VoiceOver
2. Windows, Chrome, JAWS
3. iOS, Safari, VoiceOver

Note: The combination of **Android, Chrome, TalkBack** is currently out of scope as we do not have a device for testing that; we will pursue Android testing later when a device is obtained.

## Use of color and color contrast checks

- [x] Test if text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] Test if text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Test non-text elements for a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Confirm color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Confirm any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Review content to ensure it does not refer to color, especially when providing user instructions (eg. "Click the blue button")
- [ ] (Mobile app only) Test color contrast ratios for all supported dark, contrast, and inverted color modes
- [ ] (Mobile app only) Test all elements for retaining their meaning for all supported dark, contrast, and inverted color modes

## axe checks

- [x] Each page or screen has been scanned using axe (results shared below)
- [ ] axe is integrated into your end-to-end testing
- [x] axe DevTools scan results are documented

## Content zoom and reflow checks

- [x] (Web only) Test page elements for readability and usability at 200% zoom
- [x] (Web only) Test page elements for readability and usability at 300% zoom
- [x] (Web only) Test page elements for readability and usability at 400% zoom
- [ ] (Mobile app only) Test page elements for readability and usability when device text is at the maximum possible size

## Keyboard navigation checks

- [x] Test each link, button, form input, checkbox, radio button, select menu, and custom element for receiving keyboard focus
- [x] Test each link, button, form input, checkbox, radio button, select menu, and custom element for responding to expected keys
- [x] Test all elements under focus for a visible focus indicator
- [x] Test the order of [Tab] stops to see if they make sense and are appropriate for completing tasks

## Mouse only/touchscreen checks

- [x] Tasks do not depend on specific keyboard actions
- [x] Actions do not depend on touch/click timings
- [x] Using a mobile device, full functionality can be used with only one finger to gesture

## Voiceover / Screen Reader Findings
- [x] Test each link, button, form input, checkbox, radio button, select menu, and custom element for inclusion in Voiceover / Screen Readers

### Screen reader checks:
- [x] Screen reader accurately announces content on the screen
- [x] All interactive elements are announced and electable
- [x] All links and buttons use descriptive text and can be understood out of context
- [x] Graphics contain appropriate and concise alt text
- [x] Dynamic content is announced with appropriate timing and context

### Voice command / speech recognition checks:

- [x] Able to complete the entire flow using default voice command settings
- [x] All interactive elements meet Label in Name requirements (accessible name and visual label begin with the same string)

**Comment by oddball-lindsay** [comment]
This canvas for ARP Testing should help you with 526ez testing: https://dsva.slack.com/docs/T03FECE8V/F0951KYDSHZ?focus_section_id=temp:C:aYJ8087955ff80b474bb74d096b2

**Comment by darbyemadewell** [comment]
axe scan results have been [uploaded to the SharePoint folder for this ticket](https://dvagov.sharepoint.com/:f:/r/sites/vaabdvro/Shared%20Documents/Accredited%20Representatives/Accessibility%20Testing%20Findings/126268%20-%20Submissions%20%26%20526EZ/axe%20Scan%20Results?csf=1&web=1&e=J4egWG). One issue was found on the 526EZ Review and Submit page and will be documented in the results spreadsheet for this ticket.

<img width="2560" height="1440" alt="Image" src="https://github.com/user-attachments/assets/cf819e6e-38ea-454f-9196-59e3222c4298" />
